docs/docs/contributing/feature-requests.md

```diff
@@ -0,0 +1,23 @@
+# Requesting new Features
+
+Given that we are currently at the beginning of the development of
+the Bloqade python interface, we are open to suggestions about what
+features would be helpful to include in future package iterations.
+If you have a request for a new feature, please open an issue on our
+GitHub page: https://github.com/QuEraComputing/bloqade-python/issues
```

Comment:
It's not a bad idea to request a format for Github issues like this here in the docs, *but* it would be nice if we had Github issues templates that reminded users of this on Github